service_definition.go

```diff
@@ -0,0 +1,149 @@
+package userconfig
```

Comment:
I assume there are no other changes in here than renaming & copying from other files.
Right?